resources/fcontext.rb

```diff
@@ -5,4 +5,4 @@
 default_action :addormodify
 attribute :file_spec, :kind_of => String, :name_attribute => true
 attribute :secontext, :kind_of => String
-attribute :filetype, :kind_of => String, :equal_to => ['a', 'f', 'd', 'c', 'b', 's', 'l', 'p']
+attribute :filetype, :kind_of => String, :default => 'a', :equal_to => ['a', 'f', 'd', 'c', 'b', 's', 'l', 'p']
```

Comment:
Did you thoroughly test this on all supported platforms? Because I still believe this isn't default behaviour, nil is in fact. If we do this, we don't allow to specify the -f -a option for semanage, but in fact revert it to nil ourselves ...